content/api-streamer/overview.textile

```diff
@@ -0,0 +1,67 @@
+---
+title: Overview
+section: api-streamer
+index: 65
+---
+
+h2(#what-is-api-streamer). What is API Streamer?
+
+"API Streamer":https://www.ably.io/api-streamer makes it easy for developers to publish realtime APIs (also known as event-driven APIs or streaming APIs). Built on <a href="https://www.ably.io/dsn">Ably's global Data Stream Network</a>, API Streamer provides an additional layer of tooling and features for deploying streaming data to the Ably network, managing data streams, and distributing those data streams to developer teams at third party organizations through realtime push and pull subscription APIs.
+
+<a href="https://www.ably.io/api-streamer">
+  <img src="/images/diagrams/api-streamer-overview.png" style="width: 100%" alt="Ably API Streamer">
+</a>
+
+h3(#why-use-api-streamer). Why use the API Streamer to build realtime APIs for your streaming data?
+
+When organizations decide to offer their data via APIs to developers, they typically start by building REST APIs. REST APIs are prolific, in a large part due to standardization allowing organizations to simply communicate with each other, synchronously, following a request/response pattern. To date, REST APIs have served developers well and are largely responsible for the catalyst that drove the API economy to emerge.
+
+However, as organizations are increasingly moving towards event-driven architectures, and need to deliver realtime experiences to their end-users, the need for realtime business-to-business data exchange is rapidly emerging. REST APIs are still mostly being used to solve realtime data sharing requirements in spite of their design not being suitable for streaming data. As a result, REST APIs are frequently being subjected to excessive polling, infrastructure is becoming overloaded, and the latencies for realtime data delivery is high.
+
+Many early adopter organizations are addressing this problem by building their own realtime APIs, typically based on WebSockets, WebHooks or a custom streaming HTTP protocol. These organizations are realizing that delivering realtime data at scale, with low latency, across the current and emerging protocols, with the necessary tooling to manage these data flows, is a significant upfront and ongoing engineering undertaking.
+
+This is the problem that the API Streamer addresses. Developers can focus on the data that needs to be shared, whilst Ably provides ready-built realtime API management tools, a global low latency and reliable distribution network, protocol interoperability and limitless scale.
+
+Here's a look at the increasing engineering effort required with time and growing customers, when building a realtime API from scratch vs when using the API Streamer:
```

Comment:
This is again quite sales-y for a docs article, perhaps worth thinking of how much we want it to be.